docs/user_guide/emummc/sd_preparation.md

```diff
@@ -17,24 +17,25 @@ Atmosphere has its own bootloader, called fusee (primary). For the purposes of t
 ### What you need
 
 !!! tip ""
-    - The latest release of [Hekate](https://github.com/CTCaer/hekate/releases/)
-        - Download the `hekate_ctcaer_(version).zip` release of hekate
+    - The latest release of <a href="https://github.com/CTCaer/Hekate/releases/" target="_blank">Hekate</a> (Download the `hekate_ctcaer_(version).zip` release of hekate)
+  
```

Comment:
Do not add blank spacing to this list. This is consistent across the file, so I'm just adding it here once.

This produces new list elements (`<ul>`) for each entry, which just looks ugly.

Reply:
Had this chat in exelix's server, fixed.